application/config/config.sample.php

```diff
@@ -19,6 +19,8 @@
 
 $config['callbook'] = "hamqth"; // Options are hamqth or qrz
 
+$config['datadir'] = null; // default to install directory
```

Comment:
`install/config/config.php` should be updated with this too. The `/install` route uses it as a blueprint for `application/config/config.php`.

I'm not exactly sure why these files are duplicated, I'd say either `application/config/config.sample.php` or `install/config/config.php` should be kept